### PR TITLE
refactor(react-query): improve status table

### PR DESCRIPTION
**Pull request by gwansikk**
# Overview

- I have set the Status Table to an appropriate size.
- If compatibility issues are detected, display a help message in a new row on how to fix them.

| 🟢  | ❌ |
| --- | --- |
|image|image|

## PR Checklist

- [x] I did below actions if need

1. I read the [Contributing Guide](https://github.com/toss/suspensive/blob/main/CONTRIBUTING.md)
2. I added documents and tests.

### CODE REVIEW

File: packages/react-query/src/scripts/utils/table.ts [hunk 1]
```diff
@@ -15,28 +15,28 @@ export function getStatusTable(currentTargetVersion: string) {
...
   const suspensiveAPIs = getTargetSuspensiveReactQueryAPIs()
   const exportAPIs = getExportAPIsWithoutSuspensive()
-  const peerDependencyDescription = exportAPIs.length > 0 ? `You can use ${exportAPIs.join(', ')}` : ''
 
   const table = new Table({
-    head: [packageJson.name, 'result', 'status', 'description'],
+    head: [packageJson.name, 'status', 'result'],
     style: { head: [] },
+    colWidths: [36, 10, 40],
+    wordWrap: true,
```
Comment (manudeli) [inline packages/react-query/src/scripts/utils/table.ts]: Cool

```suggestion
    colWidths: [36, 10, 40],
    wordWrap: true,
```

**Review by manudeli — APPROVED** [review]
@gwansikk I added commit to this branch to simplify table as below. if you disagree, revert commit please

image

**Comment by gwansikk** [comment]
> @gwansikk I added commit to this branch to simplify table as below. if you disagree, revert commit please
> 
> image

Cool! It looks even better than before!!

File: .changeset/early-suits-watch.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@suspensive/react-query": patch
+---
+
+refactor(react-query): improve status table
```
Comment (manudeli) [inline .changeset/early-suits-watch.md:5]:
```suggestion
fix(react-query): improve status table
```

Comment (manudeli) [reply .changeset/early-suits-watch.md]: @gwansikk refactor should be same result for users with before

**Pull request merged**